Lib/ldap/controls/ppolicy.py

```diff
@@ -62,17 +62,25 @@ class PasswordPolicyResponseValue(univ.Sequence):
 
 
 class PasswordPolicyControl(ValueLessRequestControl,ResponseControl):
+  """
+  Indicates the errors and warnings about the password policy.
+  """
   controlType = '1.3.6.1.4.1.42.2.27.8.5.1'
 
   def __init__(self,criticality=False):
     self.criticality = criticality
 
-  def decodeControlValue(self,encodedControlValue):
-    ppolicyValue,_ = decoder.decode(encodedControlValue,asn1Spec=PasswordPolicyResponseValue())
     self.timeBeforeExpiration = None
+    """The time before the password expires."""
```

Comment:
This style of inline comments is uncommon and won't affect documentation. We don't use ``epydoc`` or ``napoleon`` markup to document attributes yet.

Reply:
I am not sure what you suggest. This is the type of inline comment sphinx uses in their samples in the link I provided. I checked the generated documentation, and it displays what I expected to.

Reply:
I see. Sphinx seems to parse the code to find these extra lines. Inline strings are ignored by Python. I don't like them because they are extra noise that district from code. Instead I would prefer to use Napoleon, epydoc, or at least ``#:`` comments.

Reply:
This should be better now.